cl/search/models.py

```diff
@@ -2389,6 +2389,22 @@ class OpinionCluster(AbstractDateTimeModel):
         blank=True,
         db_index=True,
     )
+    arguments = models.TextField(
+        help_text="The attorney(s) and legal arguments presented as HTML text. "
+        "This is primarily seen in older opinions and can contain "
+        "case law cited and arguments presented to the judges.",
+        blank=True,
+    )
+    head_matter_footnotes = models.TextField(
+        help_text="Footnotes that appear before an opinion or at the conclusion "
+        "but that are not part of the opinion",
+        blank=True,
+    )
+    book_images = models.TextField(
+        help_text="Book images containing cluster content "
+        "(for example handwritten recusals) as HTML base64 images",
+        blank=True,
+    )
```

Comment:
I'm quite hesitant to put images in the DB. We've never done so before and it's usually an anti-pattern. Any reason not to do our usual S3 approach here? 

Second question: OK, so we have an image associated with the opinion. Is it possible to have more than one? How do we know where to put that image in the opinion?

Reply:
yes it is possible to have more than one - but Im not sure we've had two outside the opinion.  I'll double check

Reply:
I'm still waiting on some clarity here:

1. Shouldn't these go in S3?
2. Might we have more than one image per opinion?
3. How does the opinion know to insert an image?
4. How does the opinion know *where* to insert an image? 

It sort of feels like maybe images should go in a joined table, but maybe I'm over thinking it?

Reply:
That field has already been removed.